modules/ROOT/pages/tutorial-manage-an-api.adoc

```diff
@@ -68,24 +59,22 @@ traits:
 ** Time Period: 1
 ** Time Unit: Minute
 
-== To Apply the Policy
-
-This procedure assumes you just finished add an SLA tier as described above, or you navigated to API Manager > API Administration. In the latter case, click an API instance, and then follow this procedure to apply a policy.
+== Applying the Policy
 
-. Choose Policies.
+. Select *Policies*.
 +
-. Click Apply New Policy, and in Select Policy, select Rate Limiting - SLA-based policy, and click Configure. Alternatively, select Rate Limiting > Configure Policy.
-. In Limits, set the number of requests, time period, and units of measurement.
-. Configure the policy on the `/users` resource of the API.
-* In Method & Resource Conditions, select Apply Configurations to Specific Methods & Resources.
-* In Methods, select GET and in URI template regex.
-* In URI Template Regex, enter `/users` to apply rate limiting only to the `/users` resource, or enter `.*` to apply rate limiting to every resource URI of the API.
-* Mule 4 use: Configure header propagation for token enforcement and rate limiting policies by checking the Expose Headers checkbox.
-* Click Apply.
+. Click *Apply New Policy*, select *Rate Limiting - SLA-based*, then click *Configure Policy*.
```

Comment:
Consider making this more than one step.
In any case, avoid using ", then" as a conjunction:
```suggestion
. Click *Apply New Policy*, select *Rate Limiting - SLA-based*, and then click *Configure Policy*.
```

Reply:
Fixed!